StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentOption+Images.swift

```diff
@@ -195,3 +195,9 @@ extension STPPaymentMethodType {
         return image.makeImage(overrideUserInterfaceStyle: forDarkBackground ? .dark : .light)
     }
 }
+
+extension String {
+    var toCardBrand: STPCardBrand? {
```

Comment:
Tiny nit, I'd leave this as `STPCard.brand(from: self)` this convenience extension seems a little unusual for a String. I don't feel strongly about it though

Reply:
It's mainly useful for the optional chaining so we can invoke it on a potentially nil string, `STPCard.brand(from: self)` takes in a non-nil `String` and returns a non-nil `STPCardBrand`, if we pass an empty string (in place of a nil string) it will give back `unknown`, when in these cases we'd want to fallback on the original card brand. To use `STPCard.brand(from: self)` directly we'd need to do some additional unwrapping/branching which isn't terrible but a bit cumbersome.

Reply:
Makes sense!